pkg/realtime/memsub.go

```diff
@@ -1,16 +1,17 @@
 package realtime
 
+import "sync/atomic"
+
 // Subscription to multiple hub channels.
 type sub struct {
 	topics []*topic
-	closed chan bool
 	send   chan *Event
+	c      uint32 // mark whether or not the sub is closed
```

Comment:
atomic does not have operations on bool. i think it is pretty common in go to use a uint32 for that purpose (see [once](https://github.com/golang/go/blob/964639cc338db650ccadeafb7424bc8ebb2c0f6c/src/sync/once.go) or [exec](https://github.com/golang/go/blob/964639cc338db650ccadeafb7424bc8ebb2c0f6c/src/os/exec.go#L29-L35)).